rest/model/commit_queue_test.go

```diff
@@ -89,7 +89,16 @@ some more lines
 	assert.Len(data.Modules, 1)
 	assert.Equal(utility.ToStringPtr("module1"), data.Modules[0].Module)
 	assert.Equal(utility.ToStringPtr("1234"), data.Modules[0].Issue)
-	assert.Equal(`this is my commit message
-some more lines
-    extra whitespace  `, data.MessageOverride)
+	assert.Equal("this is my commit message some more lines extra whitespace", data.MessageOverride)
```

Comment:
Can we add a test with a long word at the end of the line to make sure that it doesn't break up words?